### PR TITLE
Added MSSQL Connection in Java Code example

### PR DESCRIPTION
**Pull request by AnderWoche**
This is my first small test post :D Please check the sytax

### CODE REVIEW

File: source/awe.rst [hunk 1]
```diff
@@ -2,4 +2,36 @@
...
\ No newline at end of file
+Unterrichtet von Herrn Katzberg
+
+#####################
+MSSQL Server Anbindung in Java
+#####################
+
+.. code-block:: java
+import java.sql.*;
+
```
Comment (pichlerer) [inline source/awe.rst:13]: There needs to be an empty line after ``.. code-block:: java``

File: source/awe.rst [hunk 2]
```diff
@@ -2,4 +2,36 @@
...
+            while(resultSet.next()) {
+                System.out.println(resultSet.getString("NAME") + " : " + resultSet.getString("Nachname"));
+            }
+
+        } catch (SQLException throwable) {
+            throwable.printStackTrace();
+        }
+    }
+}
```
Comment (pichlerer) [inline source/awe.rst:36]: This whole block has to be indented

**Comment by pichlerer** [comment]
Thanks for contributing! You can use https://livesphinx.herokuapp.com/ to test your changes. I commented on what has to be changed

File: source/awe.rst (hunk 2 above)
Comment (AnderWoche) [reply source/awe.rst:36]: should we continue speak in english or german?

Comment (pichlerer) [reply source/awe.rst:36]: I don't care to be honest :D

**Comment by AnderWoche** [comment]
I have fixed the syntax errors

**Pull request merged**